src/Toolkit/A.php

```diff
@@ -149,6 +164,28 @@ public static function get(
 		return $default;
 	}
 
+	/**
+	 * Checks in array has a value
+	 *
+	 * @param array $array
+	 * @return bool
+	 */
+	public static function has(array $array, $value): bool
+	{
+		return in_array($value, $array);
+	}
+
+	/**
+	 * Checks in array includes a value (alias for A::has)
+	 *
+	 * @param array $array
+	 * @return bool
+	 */
+	public static function includes(array $array, mixed $value): bool
+	{
+		return A::has($array, $value);
+	}
```

Comment:
I don't think needed an alias method.

Reply:
Convinced myself it's a good idea for people coming from js land. Will remove.